packages/react-utilities/README.md

```diff
@@ -1,17 +1,31 @@
 # @fluentui/react-utilities
 
-**React Utilities for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui) components**
+Utilities for providing and consuming window/document objects.
```

Comment:
some unexpected changes in `react-utilities` ,  which should go in `window-provider`

Reply:
Fixed.